nl.hjdskes.gcolor3.json

```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "nl.hjdskes.gcolor3",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.28",
+    "sdk": "org.gnome.Sdk",
+    "command": "gcolor3",
+
```

Comment:
There seems to be a misunderstanding here. When I said remove the white space I meant all of these empty lines you have.

Reply:
Ah, misunderstanding indeed.